coffee-shop/src/pages/Order/Order.jsx

```diff
@@ -1,12 +1,14 @@
 import React from "react";
 import CustomizedRadioButton from "./components/CustomizedRadioButton/CustomizedRadioButton";
+import Button from "../../genericComponents/Button/Button";
```

Comment:
as you added index in button component you just import it from the index like that 
`import Button from "../../genericComponents/Button/";`